Resources/doc/index.markdown

```diff
@@ -4,7 +4,7 @@
 
 ``` yaml
 "require": {
-    "lunetics/locale-bundle": "2.2.*",
+    "lunetics/locale-bundle": "2.4.*",
```

Comment:
the best would be to recommend `composer require lunetics/locale-bundle`. then composer will pick the last stable version itself and update the composer.json file.